src/emqttd_app.erl

```diff
@@ -32,9 +23,9 @@
 %% Application callbacks
 -export([start/2, stop/1]).
 
-%%%=============================================================================
-%%% Application callbacks
-%%%=============================================================================
+%%--------------------------------------------------------------------
+%% Application callbacks
+%%--------------------------------------------------------------------
 
 -spec start(StartType, StartArgs) -> {ok, pid()} | {ok, pid(), State} | {error, Reason} when 
```

Comment:
Line 30 is too long: -spec start(StartType, StartArgs) -> {ok, pid()} | {ok, pid(), State} | {error, Reason} when .